test/LetConst/constreassign1.baseline

```diff
@@ -1,8 +1,8 @@
 3
-ReferenceError: Assignment to const
+TypeError: Assignment to const

```

Comment:
Same is true with this file - this test can easily be converted to use the UnitTestFramework, instead of dumping the baseline. I'll leave this up to you to make a call on this.